k8s-install/scripts/install-cni.sh

```diff
@@ -51,17 +85,30 @@ sed -i s/__KUBERNETES_SERVICE_PORT__/${KUBERNETES_SERVICE_PORT:-}/g calico.conf.
 sed -i s/__SERVICEACCOUNT_TOKEN__/${SERVICEACCOUNT_TOKEN:-}/g calico.conf.tmp
 sed -i s/__KUBECONFIG_FILENAME__/calico-kubeconfig/g calico.conf.tmp
 
-# Use alternative command character "~", since ETCD_ENDPOINTS includes a "/".
+# Use alternative command character "~", since these include a "/".
+sed -i s~__KUBECONFIG_FILEPATH__~${HOST_CNI_NET_DIR}/calico-kubeconfig~g calico.conf.tmp
+sed -i s~__ETCD_CERT_FILE__~${CNI_CONF_ETCD_CERT:-}~g calico.conf.tmp
+sed -i s~__ETCD_KEY_FILE__~${CNI_CONF_ETCD_KEY:-}~g calico.conf.tmp
+sed -i s~__ETCD_CA_CERT_FILE__~${CNI_CONF_ETCD_CA:-}~g calico.conf.tmp
 sed -i s~__ETCD_ENDPOINTS__~${ETCD_ENDPOINTS:-}~g calico.conf.tmp
 
 # Move the temporary CNI config into place.
-mv calico.conf.tmp /host/etc/cni/net.d/${CNI_CONF_NAME:-10-calico.conf}
-echo "Wrote CNI config: $(cat /host/etc/cni/net.d/10-calico.conf)"
+FILENAME=${CNI_CONF_NAME:-10-calico.conf}
+mv calico.conf.tmp /host/etc/cni/net.d/${FILENAME}
```

Comment:
Hm.  I wanted to leave this here since I see a future where we might need to copy this to one of multiple potential mount locations.  I'm not really too worried about this.  Should I be?